alfresco-solrapi-client/alfresco-solrapi-client-spring/src/main/java/eu/xenit/alfresco/solrapi/client/spring/SolrApiSpringClient.java

```diff
@@ -186,6 +193,77 @@ public SolrTransactions getTransactions(Long fromCommitTime, Long minTxnId, Long
         return result.getBody().getNodes();
     }
 
+    @Override
+    public GetTextContentResponse getTextContent(Long nodeId, String propertyQName) {
+        UriComponentsBuilder uriBuilder = UriComponentsBuilder.fromHttpUrl(url).path("/textContent");
+        conditionalQueryParam(uriBuilder, "nodeId", nodeId, Objects::nonNull);
+        conditionalQueryParam(uriBuilder, "propertyQName", propertyQName, Objects::nonNull);
+
+        ResponseEntity<Resource> exchange = restTemplate
+                .exchange(uriBuilder.toUriString(), HttpMethod.GET, null, Resource.class);
+
+        GetTextContentResponse content = new GetTextContentResponse();
+        try {
+            if(exchange.getBody()!=null)
+                content.setContent(exchange.getBody().getInputStream());
+            if(exchange.getHeaders().getContentType()!=null && exchange.getHeaders().getContentType().getCharset()!=null)
+                content.setContentEncoding(exchange.getHeaders().getContentType().getCharset().displayName());
+            List<String> transformStatus = exchange.getHeaders().get("X-Alfresco-transformStatus");
+            if(transformStatus!=null && !transformStatus.isEmpty())
+                content.setTransformStatusStr(transformStatus.get(0));
+            List<String> transformDuration = exchange.getHeaders().get("X-Alfresco-transformDuration");
+            if(transformDuration!=null )
```

Comment:
the formatting seems off in this file

Reply:
Not sure what the problem is here, intellij reports no indentation problem.